scripts/ci/_utils.sh

```diff
@@ -1721,3 +1748,436 @@ function run_prepare_backport_readme() {
 function get_airflow_version_from_production_image() {
      docker run --entrypoint /bin/bash "${AIRFLOW_PROD_IMAGE}" -c 'echo "${AIRFLOW_VERSION}"'
 }
+
+function dump_kind_logs() {
```

Comment:
Let's move all KinD functions into a "_kind.sh" that can be imported by "_utils.sh". It's really hard to find functions with the current setup.

Reply:
While I am super happy to do it - I think I will do it in a separate change (right after that one is merged). I used to have such split (I believe all the related functions should be grouped in separate files - basically treated as modules in python) before but some other commiters complained that they should be combined. I am glad you support me with that :).

But this change is  already hue and I think it's better to do it separately.